src/styles/cards.less

```diff
@@ -384,7 +384,13 @@
 
             &:before {
                 border-left-color: @draggable-card-active-border;
-                filter: drop-shadow(3px 0 2px @draggable-card-active-shadow);
+                /** RTLCSS does not adjust this setting, instead we add specific pseudo selector configurations **/
+                :dir(ltr) {
+                    filter: drop-shadow(3px 0 2px @draggable-card-active-shadow);
+                }
+                :dir(rtl) {
+                    filter: drop-shadow(-3px 0 2px @draggable-card-active-shadow);
+                }
```

Comment:
`:dir` seems not to be well-supported:

https://caniuse.com/?search=%3Adir

Something like this might work:
```css
filter: drop-shadow(3px 0 2px @draggable-card-active-shadow)/*rtl:drop-shadow(-3px 0 2px @draggable-card-active-shadow)*/;
```